oaknational/Oak-Web-Application
# Feat/lesq 642/eyfs transcripts [LESQ-642]

## Description

Music year: 1974


- update lmv_lesson_overview
- update schema for transcriptSentences 
- update transcript component

## How to test

Should be no changes until EYFS published (test on migraiton env)

1. Go to https://deploy-preview-2261--oak-web-application.netlify.thenational.academy
2. Click on legacy lesson and new lesson
3. You should see transcripts still work
